ext/net/ops_tls.rs

```diff
@@ -813,14 +813,10 @@ where
     .try_borrow::<UnsafelyIgnoreCertificateErrors>()
     .and_then(|it| it.0.clone());
 
-  // TODO(@justinmchase): Ideally the certificate store is created once
-  // and not cloned. The store should be wrapped in Arc<T> to reduce
-  // copying memory unnecessarily.
```

Comment:
I tried wrapping the RootCertStore in an `Arc`, but some of the other code we don't control that this is passed to requires a `RootCertStore`.

Reply:
That's correct and that was my main problem with it. I wonder if we should open a PR to these libraries to accept `Arc<RootCertStore>`

Reply:
Maybe after we get a benchmark.